lib/senkyoshi/models/wikipage.rb

```diff
@@ -19,7 +19,7 @@ def canvas_conversion(course, resources)
         # Add page links to page body
         @files.each do |file|
           if canvas_file = course.files.detect { |f| f.identifier == file.name }
-            page.body << file.canvas_conversion(canvas_file)
+            page.body << file.canvas_conversion(resources, canvas_file)
           else
             page.body << "<p>File: " + file.linkname +
```

Comment:
Can you update this as well while you are in here?
```ruby
page.body <<
  "<p>File: #{file.linkname} -- doesn't exist in blackboard</p>"
```

Just because it is bugging me. :]